pkgs/os-specific/linux/disk-indicator/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchgit, libX11 }:
 
 stdenv.mkDerivation {
-  name = "disk-indicator";
+  name = "disk-indicator-git-2014-05-19";
```

Comment:
This causes the package name to be `disk-indicator-git`, which is probably not what we want.